apps/web/src/composables/environment.ts

```diff
@@ -3,10 +3,24 @@ export default function useEnvironment() {
     const ledgerType = import.meta.env.PUBLIC_SPECULOS_PORT ? 'speculos' : 'usb'
     const speculosURL = import.meta.env.PUBLIC_SPECULOS_PORT ? `http://127.0.0.1:${import.meta.env.PUBLIC_SPECULOS_PORT}` : undefined
     const walletConnectURL = 'https://bridge.walletconnect.org'
+    
```

Comment:
Can add:
```ts
const authBaseURL = _getAuthBaseURL()
```
And export this instead of _getAuthBaseURL. 

Can you also make that casing switch `_getAuthBaseUrl` to `_getAuthBaseURL`?

Reply:
Done.